pkgs/by-name/gl/glamoroustoolkit/package.nix

```diff
@@ -46,10 +49,22 @@ stdenv.mkDerivation (finalAttrs: {
   dontPatchELF = true;
   dontStrip = true;
 
+  desktopItems = with finalAttrs; [
+    (makeDesktopItem {
+      name = pname;
+      desktopName = "GlamorousToolkit";
+      exec = "GlamorousToolkit";
+      icon = "GlamorousToolkit";
+    })
+  ];
+
   installPhase = ''
     runHook preInstall
 
     install -d $out/bin $out/lib
+    mkdir -p $out/share/icons/hicolor/scalable/apps
```

Comment:
Hi @qbit ,
Thanks for the PR!
When I first submitted the original package I used `mkdir` in the same way and was asked to replace it with `install`, so for consistency, would you mind replacing the `mkdir` as well, i.e.:

```
install -d $out/bin $out/lib $out/share/icons/hicolor/scalable/apps
```

Reply:
ope, i saw the install but read it as copying bin into lib.. for some reason :D 

fixed!